src/BlackSlope.Api.Common/Middleware/Correlation/CurrentCorrelationIdService.cs

```diff
@@ -13,10 +13,10 @@ public CorrelationId Current()
                 return new CorrelationId(_correlationId.Value);
             }
 
-            throw new InvalidOperationException($"CorrelationId has not been set");
+            throw new InvalidOperationException(FormattableString.Invariant($"CorrelationId has not been set"));
```

Comment:
String interpolation seems unnecessary here.

Reply:
yeah, will remove $ as it's not necessary but kept Invariant call so CA1303 warning is not triggered.